doc/jetty-config.md

```diff
@@ -39,6 +39,16 @@ that exceeds this value, Jetty will return an HTTP 413 Error response. This
 defaults to 8192 bytes, and only needs to be configured if an exceedingly large
 header is being sent in an HTTP Request.
 
+### `so-linger-seconds`
+
+This sets the TCP SO_LINGER time, in seconds, that the webserver uses for
+underlying socket connections.  Values less than 0 result in SO_LINGER
+being disabled.  Defaults to -1, i.e., "disabled".  For a more detailed
+description of what it means to have SO_LINGER disabled vs. enabled for some
+number of seconds, see http://man7.org/linux/man-pages/man7/socket.7.html.  Note
+that the effect of setting this option may vary depending upon the operating
+system's underlying implementation.
+
```

Comment:
This is a great description of what the configuration setting does, but I'm left wondering what the purpose of the setting is?  Could the description be updated to describe why someone may want to tune this setting?